packages/app-store/huddle01video/api/add.ts

```diff
@@ -40,5 +41,5 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return res.status(500);
   }
   // need to return a json object with the response status
-  return res.status(200).json({ url: "/apps/installed" });
+  return res.status(200).json({ url: getInstalledAppPath({ variant: "conferencing", slug: "huddle01" }) });
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.